### ISSUE TITLE
cryptoSign vs cryptoSignOpen

### ISSUE THREAD

**Issue by phxql** [terl/lazysodium-java#61]
Hi,

```
String cryptoSign(String message, String secretKey) throws SodiumException;
```

takes a `String` secret key, while 

```
String cryptoSignOpen(String signedMessage, Key publicKey);
```

takes a `Key` public key.

Maybe add an overload 

```
String cryptoSign(String message, Key secretKey) throws SodiumException;
```

to the API? Otherwise i have to think about how to get a `String` from `Key`. Doable, but not pretty.

Thanks!

**Comment by gurpreet-** [comment]
Hi @phxql,

Thank you for your request to add a new overloaded method. I've added the `cryptoSign(String message, Key secretKey)` method and it's available in version `4.2.4`.

Enjoy!

**Comment by phxql** [comment]
Great, thanks a lot!
